subprojects/docs/src/docs/userguide/gradleDaemon.adoc

```diff
@@ -0,0 +1,224 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[gradle_daemon]]
+== The Gradle Daemon
+
+From Wikipedia…
+
+++++
+<blockquote>A daemon is a computer program that runs as a background process, rather than being under the direct control of an interactive user.</blockquote>
+++++
+
+Gradle runs on the Java Virtual Machine (JVM) and uses several supporting libraries that require a non-trivial initialization time. As a result, it can sometimes seem a little slow to start. The solution to this problem is the Gradle _Daemon_: a long-lived background process that executes your builds much more quickly than would otherwise be the case. We accomplish this by avoiding the expensive bootstrapping process as well as leveraging caching, by keeping data about your project in memory. Running Gradle builds with the Daemon is no different than without. Simply configure whether you want to use it or not - everything else is handled transparently by Gradle.
+
+
+[[sec:why_the_daemon]]
+=== Why the Gradle Daemon is important for performance
+
+The Daemon is a long-lived process, so not only are we able to avoid the cost of JVM startup for every build, but we are able to cache information about project structure, files, tasks, and more in memory.
+
+The reasoning is simple: improve build speed by reusing computations from previous builds. However, the benefits are dramatic: we typically measure build times reduced by 15-75% on subsequent builds. We recommend profiling your build by using `--profile` to get a sense of how much impact the Gradle Daemon can have for you.
+
+The Gradle Daemon is enabled by default starting with Gradle 3.0, so you don't have to do anything to benefit from it.
+
+[[sec:status]]
+=== Running Daemon Status
+
+To get a list of running Gradle Daemons and their statuses use the `--status` command.
+
+Sample output:
+[source]
+----
+
```

Comment:
I removed them. The automatic conversion seems to add those newlines and I didn't care to remove them up to now.